### PR TITLE
Created search and filters components for articles

### PR DESCRIPTION
**Pull request by Juli11V**
Create a filters components.
Create a search components.
Create an articles components.
Create service for filterArticles.

### CODE REVIEW

File: src/components/AdminContextAreaManagement/AdminContentArea/AdminFiltersContentArea/AdminFiltersContentArea.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+            <AdminFilterContentArea  defValue="All Teams" components ={teams} handleChange={handleTeamFilterChange}/>
+            <AdminFilterContentArea  defValue="All" components ={articlesStatus} handleChange={handleStatusFilterChange}/>
+     </div>
+             <AdminArticlesContentArea articles = {articles} />
+        </div>
+    );
+}
+
+export default AdminFiltersContentArea;
+
```
Comment (markoparfeniuk) [inline src/components/AdminContextAreaManagement/AdminContentArea/AdminFiltersContentArea/AdminFiltersContentArea.js]: 177-181 extra blank lines

File: src/components/AdminContextAreaManagement/AdminContentArea/AdminFiltersContentArea/AdminFiltersContentArea.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+import { useAuth0 } from "@auth0/auth0-react";
+import FilterArticlesService from "../../../../services/FilterArticlesService/FilterArticlesService";
+import SubcategoryService from "../../../../services/SubcategoryService/SubcategoryService";
+import TeamService from "../../../../services/TeamService/TeamService";
+
+
+function AdminFiltersContentArea()  {
+
+    const categoryId = "c4ac0c1d-26fe-4877-b81d-08da90d6d7d1";
+
```
Comment (markoparfeniuk) [inline src/components/AdminContextAreaManagement/AdminContentArea/AdminFiltersContentArea/AdminFiltersContentArea.js]: Could you please remove the blanks in lines 15-52?

File: src/components/AdminContextAreaManagement/AdminContentArea/AdminContentArea.js [hunk 3]
```diff
@@ -1,8 +1,13 @@
 import React from "react"
 import "./AdminContentArea.scss"
+import AdminFiltersContentArea from "./AdminFiltersContentArea/AdminFiltersContentArea";
+
```
Comment (markoparfeniuk) [inline src/components/AdminContextAreaManagement/AdminContentArea/AdminContentArea.js:4]: Here is an empty line

File: src/components/AdminContextAreaManagement/AdminContentArea/AdminSearchContentArea/AdminSearchContentArea.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import React, {useState} from "react"
+import "./AdminSearchContentArea.scss"
+import {BiSearch} from "react-icons/bi";
+
+
```
Comment (markoparfeniuk) [inline src/components/AdminContextAreaManagement/AdminContentArea/AdminSearchContentArea/AdminSearchContentArea.js:5]: Here is another blank line :)

File: src/components/AdminContextAreaManagement/AdminContentArea/AdminSearchContentArea/AdminSearchContentArea.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+            articles.forEach((article) => {
+                let tempArticle = article.headline.toLowerCase().indexOf(event.target.value.toLowerCase()) !== -1;
+                if(tempArticle) {
+                    updatedArticles.push(article);
+                }
+            });
+            handleChangeArticles(updatedArticles, true);
+        } else { handleChangeArticles(articles, false);}
+    }
+
```
Comment (markoparfeniuk) [inline src/components/AdminContextAreaManagement/AdminContentArea/AdminSearchContentArea/AdminSearchContentArea.js:23]: And here is too

File: src/services/TeamService/TeamService.js [hunk 6]
```diff
@@ -28,6 +28,12 @@ export default class TeamService {
         );
     }
```
Comment (markoparfeniuk) [inline src/services/TeamService/TeamService.js:32]: And the last one :)

**Review by markoparfeniuk — CHANGES_REQUESTED** [review]

**Review by markoparfeniuk — APPROVED** [review]

**Review by NotJediYet — APPROVED** [review]

**Review by NotJediYet — APPROVED** [review]

**Pull request closed without merging**